TASK/TaskBoatMission.md

```diff
@@ -5,10 +5,12 @@ ns: TASK
 
 ```c
 // 0x15C86013127CE63F 0x5865B031
-void TASK_BOAT_MISSION(Ped pedDriver, Vehicle boat, Any p2, Any p3, float x, float y, float z, Any p7, float maxSpeed, int drivingStyle, float p10, Any p11);
+cs_type(Any) void TASK_BOAT_MISSION(Ped ped, Vehicle boat, Ped pedTarget, Vehicle vehicleTarget, float x, float y, float z, int missionType, float speed, int drivingStyle, float radius, int missionFlags);
```

Comment:
```suggestion
void TASK_BOAT_MISSION(Ped ped, Vehicle boat, Ped pedTarget, Vehicle vehicleTarget, float x, float y, float z, int missionType, float speed, int drivingStyle, float radius, int missionFlags);
```
You don't change the native return type here so `cs_type(Any)` is not needed, the arguments you do change (Any -> Ped and Any -> Vehicle) are already valid since theyre the same underlying type